### PR TITLE
Todo list

### PR DESCRIPTION
**Pull request by Odongo006**
Hello, in this project I managed tp complete the tasks listed below: -

- Implement a function for adding new task
- Implement a function for deleting a task
- Implement a function for editing task descriptions
- Save to-do list changes to local storage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,121 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microcerse] <a name="about-project"></a>
+
+**[Awesome Books ES6]**
```
Comment (ha-manel) [inline README.md]: [Optional]

- [ ] Kindly edit the project title, it's todo list and not awesome books 😅

File: README.md [hunk 2]
```diff
@@ -1 +1,121 @@
...
+use the git clone command to clone this repository or download ZIP
+
+### Install
+
+Install this project with:
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/Odongo006/My-Portfolio-Project.git
```
Comment (ha-manel) [inline README.md]: [Optional]

- [ ] Kindly use the right cloning link, this one is for the portfolio project 😅

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @Odongo006 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* The app is working nicely 💯 

## Required Changes:

- [ ] Kindly add the MIT file to your project because the link in your README file is broken 👍 

## Optional:

- [ ] Please consider using best practices when you write commit messages, they should be capitalized sentences in the imperative form, example: "Add linters" instead of "added linters" or "linters are added" ... etc 👍 

_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Odongo006) [reply README.md]: funny, who forgets to edit the title 😆

File: README.md [hunk 3]
```diff
@@ -1 +1,121 @@
...
+
+- [Microverse](https://www.microverse.org/)
+- Code Reviewers
+- Coding Partners
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (FrederickMih) [inline README.md:121]: Capture

- Kindly edit your license file. Information is obsolete. This is not professional

**Review by FrederickMih — APPROVED** [review]
## Hi 🙋 @Odongo006 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- PR is professional 👍 
- Commit messages now followed best practices 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**